addon/hint/show-hint.js

```diff
@@ -379,12 +379,15 @@
     },
 
     scrollToActive: function() {
-      var node = this.hints.childNodes[this.selectedHint]
+      var margin = this.completion.options.scrollMargin || 0;
+      if (! Number.isInteger(margin)) margin = 0;
+      var node1 = this.hints.childNodes[Math.max(0, this.selectedHint-margin)];
+      var node2 = this.hints.childNodes[Math.min(this.data.list.length-1, this.selectedHint+margin)];
```

Comment:
(Could you follow the whitespace-around-operators style from the surrounding code?)

Reply:
done